OracleDatabase/SingleInstance/dockerfiles/buildDockerImage.sh

```diff
@@ -112,6 +112,9 @@ elif [ $EXPRESS -eq 1 ]; then
   elif [ "$VERSION" == "11.2.0.2" ]; then
     EDITION="xe"
     DOCKEROPS="--shm-size=1G $DOCKEROPS";
+  elif [ "$VERSION" == "18.4.0" ]; then
+    EDITION="xe"
+    DOCKEROPS="--shm-size=1G $DOCKEROPS";
```

Comment:
For a proper execution of the embedded JVM this is a better SHM configuration:

> -v /dev/shm --tmpfs /dev/shm:rw,nosuid,nodev,exec,size=1g

if some Java code try to use large SHM memory space during setup the OJVM experiment memory exceptions.

Reply:
The `--shm-size` was only needed because 11g XE used Automatic Memory Management, i.e. `/dev/shm` for SGA+PGA. This is no longer the case, so this option doesn't need to be passed on for the image creation.

Indeed for OJVM and other components users should mount /dev/shm themselves with the desired size.

Reply:
Should I take some additional action on this change?
Marcelo.

Reply:
Yes, the `--shm-size` needs to go. Actually you could just remove the whole `if` branch and change the one above to `18.4.0` instead.

Reply:
`--shm-size` is still not gone.